.github/workflows/cilium-integration-tests.yaml

```diff
@@ -0,0 +1,155 @@
+name: Cilium Integration Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request_target: {}
+  issue_comment:
+    types:
+      - created
+env:
+  KIND_VERSION: v0.18.0
+  CILIUM_REPO_REF: master
+  CILIUM_CLI_TAG: latest
+
+jobs:
+  cilium-connectivity-tests:
+    timeout-minutes: 360
+    name: Cilium Connectivity Tests
+    runs-on: ubuntu-latest-64-cores-256gb
+    steps:
+      - name: Prepare variables for pushing to master
+        if: github.event_name == 'push'
```

Comment:
do we need to check for base ref here i.e. branch is master ?

Reply:
yes, because otherwise we might trigger the workflow twice when pushing to a branch with a corresponding PR (push & PR sync event)

Reply:
@sayboras we might need to adapt this one once we branch new release branches from master

Reply:
actually it's also covered by the branch filter in the "on push" event definition. going to remove it.